samples/sensor/lsm6dsl/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _lsm6dsl:
+
+LSM6DSL 3D accelerometer and 3D gyroscope
+#########################################
+
+Overview
+********
+
+A simple sensor application that demonstrates how to read values from the
+LSM6DSL accelerometer and gyroscope sensor.
+
+Building and Running
+********************
+
+This project outputs sensor data to the console. It requires an LSM6DSL
+sensor, which is present on the :ref:`disco_l475_iot1` board.
+
+.. code-block:: console
+
+   $ cd samples/sensors/lsm6dsl
+   $ make BOARD=disco_l475_iot1
+
+Sample Output
+=============
+
+.. code-block:: console
+
+   Angular rate in millidegrees per sec:
+   X=     490.0 Y=   -1470.0 Z=   -1260.0
+
+   Linear acceleration in mg:
+   X=     -18.1 Y=       8.4 Z=    1034.4
```

Comment:
The sensor interface defines acceleration in m/s^2 not mg, and angular rate in rad/s not mdeg/s. Since you don't do any unit conversion in the sample app, I think there may be an issue in the sensor driver producing data with the wrong units.